package-versions/grouping-and-overrides/grouping-and-overrides.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: default
+title: Grouping and Overrides
+parent: Package Versions
+has_children: true
+nav_order: 2
+---
+
+# Grouping and Overrides
+
+Managing package versions across large numbers of complex projects may benefit from organizing `<PackageVersion>` into named groups of packages.
+
+Additionally, the usage of named groups of `<PackageVersion>` can allow individual projects to use different versions than the primary versions used globally.
+
+## Introduction
+
+In the most simple case, a `<PackageVersion>` assigned to a named group can be used to separate all of the versions in a project.
+
+#### Directory.Packages.props
+```xml
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <ItemGroup>
+        <PackageVersion Include="foo" Version="1.1.1" />
+        <PackageVersion Include="foo" Version="2.2.2" CentralManagementGroup="A" />
+    </ItemGroup>
+</Project>
+```
+
+#### project.csproj
+```xml
+<Project>
+  <ItemGroup>
+    <!-- Version 2.2.2 -->
+    <PackageReference Include="foo" CentralManagementGroup="A" />
+  </ItemGroup>
+</Project>
+```
+
+In this case, version 2.2.2 of `foo` is imported because it belongs to `CentralManagementGroup="A"`.
+
+## Managing `<PackageVersion>` Across Projects
+
+In order to preserve central management, `<PackageVersion>` should only be defined within your `Directory.Packages.props` file.
+
+While MSBuild will allow you to define `<PackageVersion>` within any MSBuild compatible file, this should be considered an anti-pattern.
```

Comment:
An anti-pattern perhaps if the goal was to keep everything strictly top-level. But in repos that contain several disparate group of projects, allowing each sub-tree of projects adjust the package versions where necessary is a nice feature rather than an anti-pattern. So is this up to us to decide?
If we decide to have a tree of `Directory.Packages.props` files, will the IDE support that?

Reply:
Also, would it be a supported scenario to have a `MySLN\Directory.Packages.props` and a `MySLN\ProductA\Directory.Packages.props` and to call `<Import Project="..\Directory.Packages.props" />` in that second file?